src/carcockpit/main.cpp

```diff
@@ -22,83 +22,20 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #include <clargs/parser.hpp>
 #include <ruisapp/application.hpp>
 
-#include "car_widget.hpp"
+#include "application.hpp"
+#include "gltf_viewer_widget.hpp"
```

Comment:
`gltf_viewer_widget` is not used in this file, remove the include

Reply:
done